### PR TITLE
Main Cleanup of Workbench release notes

### PR DESCRIPTION
**Pull request by sf1919**
Reordered release notes to group related work, added links, spelling & grammar check, added Workspace Calculator image

**Description of work.**
The main refactoring the the Workbench release notes ahead of release of 6.2. Includes:-

- Reordering of notes where appropriate
- Adding/fixing links
- Spelling & Grammar check
- Adding an image for the Workspace Calculator

**To test:**

- remove ```build/docs/doctrees```
- build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 1]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
 
-- Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See :ref:`Superplot documentation <WorkbenchSuperplot>` for more information.
+- **Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See** :ref:`Superplot documentation <WorkbenchSuperplot>` **for more information.**
 
 .. figure:: ../../images/superplot_1.png
     :width: 500px
     :align: center
 
-- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``
+- **New widget and workbench plugin:** :ref:`WorkbenchWorkspaceCalculator` **, allows to perform binary operations and scaling by a floating number on workspaces;
```
Comment (StephenSmith25) [inline docs/source/release/v6.2.0/mantidworkbench.rst:17]: Missing a word here `New widget and workbench plugin: :ref:WorkbenchWorkspaceCalculator, allows (you/the user/users)`

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 2]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
 - Plot legends can be shown or hidden from the plot context menu.
-- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
 - The plot config dialog notifies the user when there has been an error applying the config to the plot, and allows them to change the config further.
-- When fitting a plot, selecting the peak type will only update the default peak shape in the settings if the "Set as global default" checkbox is ticked.
-- Added help button to the sliceviewer
-- SliceViewer can toggle between different scales again without any issue.
-- SliceViewer uses a more visible divider between the main data view and the peaks table view.
+- The list of eligible workspaces in the `WorkspaceSelector` can now be sorted by name.
+- Added tooltips to all the widgets in the :ref:`sliceviewer`. Please contact the developers if any are missing.
+- Script editor function call tip support for Numpy 1.21.
```
Comment (StephenSmith25) [inline docs/source/release/v6.2.0/mantidworkbench.rst:31]: I feel like just saying code completion support covers everything, but maybe not.
```suggestion
- Script editor code completion support for Numpy 1.21.
```

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 3]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
+- The visibility of a component parameter in the Pick tab of the :ref:`InstrumentViewer` is now steered by the 'visible' attribute of a parameter in :ref:`IPF <InstrumentParameterFile>`.
+- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
+- Added help button to the :ref:`sliceviewer`.
```
Comment (StephenSmith25) [inline docs/source/release/v6.2.0/mantidworkbench.rst:34]:
```suggestion
- Added a help button to the :ref:`sliceviewer`.
```

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 4]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
+- :ref:`sliceviewer` uses a more visible divider between the main data view and the peaks table view.
 
 Bugfixes
 --------
 
-- "Grid" checkbox in "Edit axis" dialog, and "Grids on/off" toolbar button will now have the correct checked state when running a plot script with major grid lines.
+- ``Grid`` checkbox in ``Edit axis`` dialog, and ``Grids on/off`` toolbar button will now have the correct checked state when running a plot script with major grid lines.
+- Fixed a bug in colorfill plots which lead to the loss of a spectrum from the resulting image.
+- Fixed a bug where removing the plot guess line in the fit browser could lead to an exception being thrown.
+- Fixed the advanced plotting dialog incorrectly laying out, causing the options to be partially occluded.
```
Comment (StephenSmith25) [inline docs/source/release/v6.2.0/mantidworkbench.rst:44]:
```suggestion
- Fixed the advanced plotting dialog incorrect layout, which caused options to be partially occluded.
```

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 5]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
-- Added help button to the sliceviewer
-- SliceViewer can toggle between different scales again without any issue.
-- SliceViewer uses a more visible divider between the main data view and the peaks table view.
+- The list of eligible workspaces in the `WorkspaceSelector` can now be sorted by name.
+- Added tooltips to all the widgets in the :ref:`sliceviewer`. Please contact the developers if any are missing.
+- Script editor function call tip support for Numpy 1.21.
+- The visibility of a component parameter in the Pick tab of the :ref:`InstrumentViewer` is now steered by the 'visible' attribute of a parameter in :ref:`IPF <InstrumentParameterFile>`.
+- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
+- Added help button to the :ref:`sliceviewer`.
+- :ref:`sliceviewer` can toggle between different scales again without any issue.
```
Comment (StephenSmith25) [inline docs/source/release/v6.2.0/mantidworkbench.rst:35]: Should this be a bugfix?

**Review by StephenSmith25 — COMMENTED** [review]
Formatting looks good, I just have a couple of wording suggestions.

**Comment by sf1919** [comment]
Thanks @StephenSmith25 . I agree with all your comments so I've made the necessary changes.

**Review by StephenSmith25 — DISMISSED** [review]
Thanks! LGTM :+1:

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 6]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
 
-- Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See :ref:`Superplot documentation <WorkbenchSuperplot>` for more information.
+- **Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See** :ref:`Superplot documentation <WorkbenchSuperplot>` **for more information.**
 
 .. figure:: ../../images/superplot_1.png
     :width: 500px
     :align: center
 
-- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``
+- **New widget and workbench plugin:** :ref:`WorkbenchWorkspaceCalculator` **, allows users to perform binary operations and scaling by a floating number on workspaces;
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:17]: This sentence doesn't read well, could you rephrase it?

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 7]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
 New and Improved
 ----------------
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:9]: Could we split new from improved? There are 2 new widgets, it's better to make them stand out.

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 8]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
-- The visibility of a component parameter in the Pick tab of the InstrumentViewer is now steered by the 'visible' atrribute of a parameter in IPF
+- When fitting a plot, selecting the peak type will only update the default peak shape in the settings if the ``Set as global default`` checkbox is ticked.
 - Plot legends can be shown or hidden from the plot context menu.
-- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
 - The plot config dialog notifies the user when there has been an error applying the config to the plot, and allows them to change the config further.
-- When fitting a plot, selecting the peak type will only update the default peak shape in the settings if the "Set as global default" checkbox is ticked.
-- Added help button to the sliceviewer
-- SliceViewer can toggle between different scales again without any issue.
-- SliceViewer uses a more visible divider between the main data view and the peaks table view.
+- The list of eligible workspaces in the `WorkspaceSelector` can now be sorted by name.
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:29]: This should be removed as the widget is new in this release.

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 9]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
+- Added tooltips to all the widgets in the :ref:`sliceviewer`. Please contact the developers if any are missing.
+- Script editor code completion support for Numpy 1.21.
+- The visibility of a component parameter in the Pick tab of the :ref:`InstrumentViewer` is now steered by the 'visible' attribute of a parameter in :ref:`IPF <InstrumentParameterFile>`.
+- ADS signal handlers are now synchronized in `WorkspaceSelector`. This reduces the probability of hard crash when interacting with the widget while a script manipulating a large number of workspaces is being run.
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:33]: This as well.

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 10]
```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
...
 - Fixed a bug where the errorbar tab in the figure options was wrongly enabled while selecting multiple curves.
-- Fixed a bug where removing the plot guess line in the fit browser could lead to an exception being thrown.
 - Fixed a bug where marker formatting options were disabled upon opening the figure options.
-- Fixed a bug where the workspace index spinbox in the fit browser wouldn't update when the user added or removed curves from the figure.
-- Fixed out of range errors in the Sliceviewer that sometimes occured whilst hovering over transposed data.
+- Fixed out of range errors in the :ref:`sliceviewer` that sometimes occurred whilst hovering over transposed data.
 - Fixed the help icon not showing on OSX and high-resolution monitors.
 - Tabbing between fields in the error reporter now works as expected, rather than jumping to a random place each time.
-- Fixed the advanced plotting dialog incorrectly laying out, causing the options to be partially occluded.
+- :ref:`sliceviewer` can toggle between different scales again without any issue.
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:60]: Can we group all the slice viewer improvements together?

**Review by gvardany — COMMENTED** [review]
Few comments to improve the readability

File: docs/source/release/v6.2.0/mantidworkbench.rst (hunk 6 above)
Comment (sf1919) [reply docs/source/release/v6.2.0/mantidworkbench.rst:17]: I've had a look at the documentation for this which is similar to this release note but perhaps gives a little more detail - 'The workspace calculator allows to quickly perform binary operations on selected workspace pairs along with simultaneous scaling by arbitrary floating point numbers, provided they are different from zero.'

Maybe I should use this in the release notes instead?

File: docs/source/release/v6.2.0/mantidworkbench.rst (hunk 8 above)
Comment (sf1919) [reply docs/source/release/v6.2.0/mantidworkbench.rst:29]: Is it? I thought the two new widgets were Superplot and the Workspace Calculator. Is the Workspace Selector the same as the Workspace Calculator? I confess I couldn't find any documentation on the Workspace Selector so wasn't sure what it was exactly.

File: docs/source/release/v6.2.0/mantidworkbench.rst (hunk 6 above)
Comment (gvardany) [reply docs/source/release/v6.2.0/mantidworkbench.rst:17]: Perhaps this is more concise:
`The workspace calculator allows to quickly perform binary operations on selected workspace pairs, optionally with scaling.`

File: docs/source/release/v6.2.0/mantidworkbench.rst (hunk 8 above)
Comment (gvardany) [reply docs/source/release/v6.2.0/mantidworkbench.rst:29]: ahh, sorry, you are right, the selector is as old as mantid

File: docs/source/release/v6.2.0/mantidworkbench.rst (hunk 9 above)
Comment (gvardany) [reply docs/source/release/v6.2.0/mantidworkbench.rst:33]: please ignore this as well

File: docs/source/release/v6.2.0/mantidworkbench.rst [hunk 11]
```diff
@@ -5,57 +5,75 @@ Mantid Workbench Changes
...
-- Fixed a bug in the Instrument Viewer causing the projection to not be updated when different axis views were selected in Full 3D.
+- Fixed a bug in the :ref:`InstrumentViewer` causing the projection to not be updated when different axis views were selected in Full 3D.
+
+Sliceviewer
+-----------
 
+Improvements
+############
+
+- Peaks can now be added or removed from a PeaksWorkspace using the :ref:`peaks overlay <sliceviewer_peaks_overlay>` in :ref:`sliceviewer`.
```
Comment (gvardany) [inline docs/source/release/v6.2.0/mantidworkbench.rst:65]: Thanks for grouping these, however, now it seems redundant to repeat :ref:`sliceviewer` in each item as they are all under one heading. Perhaps sublists would be better. The topmost item can say `Several improvements have been done in :ref:`sliceviewer`. Then the sub-items can just enumerate them. Peaks overlay can, of course, be referenced as it is a different link.

**Comment by martyngigg** [comment]
@gvardany This keeps attracting merge conflicts so I'd like to get this merged and we make the changes in a separate PR?
